package.json

```diff
@@ -33,7 +33,7 @@
     "build": "react-scripts build",
     "lint": "npm-run-all -c lint:*",
     "lint:eslint": "eslint 'src/**/*.{js,jsx}'",
-    "lint:stylelint": "stylelint src/**/*.scss",
+    "lint:stylelint": "stylelint 'src/**/*.scss'",
```

Comment:
Globs must be quoted for them to work correctly:

```
Usage: stylelint [input] [options]
Input: Files(s), glob(s), or nothing to use stdin.
  If an input argument is wrapped in quotation marks, it will be passed to
  globby for cross-platform glob support. node_modules and
  bower_components are always ignored. You can also pass no input and use
  stdin, instead.
```